### PR TITLE
correct supported types table to support any type

### PR DESCRIPTION
**Pull request by moAlobaidi**
This PR closes this [issue](https://github.com/AleoHQ/techops/issues/48).

### CODE REVIEW

**Comment by miazn** [comment]
couldn't tell for sure, but does this also need to specify the finalize context? [source](https://github.com/AleoHQ/techops/issues/48#issuecomment-1601395071)

**Comment by moAlobaidi** [comment]
> 

Yup! The description [here](https://github.com/AleoHQ/welcome/blob/c520e7905a9fd88b054c854ec2112f068bb43667/documentation/aleo/04_opcodes.md#description-46) specifies that.

I can add a note at the bottom to make it clear if needed though.

**Comment by miazn** [comment]
> @miazn Yup! The description [here](https://github.com/AleoHQ/welcome/blob/c520e7905a9fd88b054c854ec2112f068bb43667/documentation/aleo/04_opcodes.md#description-46) specifies that.
> 
> I can add a note at the bottom to make it clear if needed though.

🤦 missed that! lgtm :D

**Review by miazn — APPROVED** [review]

File: documentation/aleo/04_opcodes.md [hunk 1]
```diff
@@ -1330,27 +1330,16 @@ The `rand.chacha` opcode is used to generate random values within the `finalize`
...
-| I32       | I32       | I32         |
-| I64       | I64       | I64         |
-| I128      | I128      | I128        |
-| U8        | U8        | U8          |
-| U16       | U16       | U16         |
-| U32       | U32       | U32         |
-| U64       | U64       | U64         |
-| U128      | U128      | U128        |
-| Scalar    | Scalar    | Scalar      |
+Magnitude can be an `Address`, `Boolean`, `Field`, `Group`, `I8`, `I16`, `I32`, `I64`, `I128`, `U8`, `U16`, `U32`, `U64`, `U128`, or `Scalar`.
```
Comment (collinc97) [inline documentation/aleo/04_opcodes.md:1338]:
```suggestion
Single can be any of the following types `Address`, `Boolean`, `Field`, `Group`, `I8`, `I16`, `I32`, `I64`, `I128`, `U8`, `U16`, `U32`, `U64`, `U128`, or `Scalar`. Composite data types such as structs and mappings are not allowed.
```

File: documentation/aleo/04_opcodes.md [hunk 2]
```diff
@@ -1330,27 +1330,16 @@ The `rand.chacha` opcode is used to generate random values within the `finalize`
...
+
+| First       | Second      | Destination |
+|-------------|-------------|-------------|
+| `Magnitude` | `Magnitude` | `Magnitude` |
```
Comment (collinc97) [inline documentation/aleo/04_opcodes.md:1342]:
```suggestion
| `Single` | `Single` | `Single` |
```

**Review by collinc97 — APPROVED** [review]
Changing "Magnitude" -> "Single" because we already define magnitude when describing the pow operation and I think "Single" is more clear (single type).

**Pull request merged**